lib/similarity_utils.py

```diff
@@ -256,9 +256,19 @@ def similarity_one_vs_all_new(data: pd.DataFrame, main_actor: List[Any], reduced
     ["nconst", "tconst", "category", "primaryName", "knownForTitles", "no_nominations_oscars_norm", "no_nominations_globes_norm",
     "no_nominations_emmy_norm", "no_films_norm", "average_films_rating_norm", "genres_code", "types_code"]
     
-    to należy zmienić wartość parametru 'reduced_dataset' na True"""
+    to należy zmienić wartość parametru 'reduced_dataset' na True;
+    metoda jest przygotowana pod dane ze zbiorów treningowego, testowego i walidacyjnego"""
```

Comment:
nie rozumiem co oznacza ten dopisek - w sensie po co on, metoda jest przygotowana pod każde dane

Reply:
Nowa metoda rankingowa korzysta z nowej metody liczącej similarity. Stąd skoro nowa metoda bazuje na nowych danych, to metoda tworząca ranking również.

Reply:
Skoro wleciała nowa metoda licząca similarity inaczej niż poprzednia, to stara metoda rankingująca nie będzie w stanie liczyć rankingu według nowej metody liczenia similarity